lib/sentry/mapping.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Sentry.Mapping do
+  @moduledoc false
+
+  def new() do
+    :ets.new(:cron, [:named_table, :set, :public, read_concurrency: true])
+  end
+
+  def insert(table, key, value) do
+    :ets.insert(table, {key, value})
```

Comment:
The table is always gonna be called the same. You could extract the name to a module attribute:

```elixir
@table :cron
```

and then you won't have to pass it around in fucntions.